src/campsite-info/campsite-info.component.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { FunctionComponent } from 'react';
 import { CampHeader } from './components/camp-header';
 import { CampImages } from './components/camp-images';
+import { SiteMap } from './components/camp-map';
```

Comment:
can we call this something like CampSiteMap instead? Only because a 'site map' has a very particular meaning for web pages.